extensions/community_extensions/skadnetwork.md

```diff
@@ -100,7 +114,7 @@ If a DSP has at least one SKAdNetworkItem in the publisher app’s `Info.plist`
         <code>skadnetids</code>
       </td>
       <td>
-        A subset of SKAdNetworkItem entries in the publisher app’s Info.plist that are relevant to the DSP. Recommended that this list not exceed 10.
+        A subset of SKAdNetworkItem entries in the publisher app’s Info.plist that are relevant to the bid request. Recommended that this list not exceed 10. Note: `skadnetids` will eventually be deprecated in favor of `BidRequest.imp.ext.skadn.skadnetlist.nonlistskadnetids`.
```

Comment:
I would remove this edit per previous feedback. There is a market use case for `skadnetids` even in the full list solution.